src/app/routes.js

```diff
@@ -76,6 +76,11 @@ export default (app, router, passport, io) => {
   analyticsRoutes(app, router, admin);
   pushRoutes(app, router);
 
+  // Provide a simple status page
+  router.get('/status', (req, res) => {
```

Comment:
I think the plan is to use 3rd party ping service like uptimerobot to check the health of the db connection, and get email notification if the connection died